src/components/Header/Header.js

```diff
@@ -39,15 +41,28 @@ export class Header extends PureComponent {
     });
   }
 
+  handleDropdownVariantsClick = (id, title) => {
+    const { addCurrentLocation } = this.props;
+
+    addCurrentLocation(id);
+    this.setState({
+      addressPlaceholder: title,
```

Comment:
placeholder exists to render some hint. I think we can just change address to the `title`